src/macros/src/lib.rs

```diff
@@ -0,0 +1,72 @@
+#![recursion_limit = "192"]
+
+extern crate proc_macro;
+extern crate syn;
+#[macro_use] extern crate quote;
+
+use proc_macro::TokenStream;
+
+
+#[proc_macro_derive(GfxVertexFormat)]
```

Comment:
is the `Gfx` prefix necessary ? (I hope there is a way to rename a macro on the user side if he has clashes)